src/index.js

```diff
@@ -0,0 +1,101 @@
+import { createInterface } from "readline";
+import { fileURLToPath } from "url";
+import { dirname } from "path";
+import up from "./nwd/up.js";
+import cd from "./nwd/cd.js";
+import ls from "./nwd/ls.js";
+import cat from "./fo/cat.js";
+import add from "./fo/add.js";
+import rn from "./fo/rn.js";
+import mv from "./fo/mv.js";
+import cp from "./fo/cp.js";
+import rm from "./fo/rm.js";
+import os from "./os/os.js";
+import hash from "./hash/hash.js";
+import compress from "./archivate/compress.js";
+import decompress from "./archivate/decompress.js";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+let currentDir = __dirname;
+const rootDir = __dirname.split("\\")[0];
+
+const user = process.env.username || "Anonymous";
+console.log(`Welcome to the File Manager, ${user}!`);
+console.log(`You are currently in ${currentDir}`);
+
+process.on("exit", function () {
+  console.log(`\nThank you for using File Manager, ${user}, goodbye!`);
+});
+
+var rl = createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const menu = function () {
+  rl.question("", function (answer) {
```

Comment:
```suggestion
  rl.question("", async function (answer) {
```
Если хочешь, чтобы асинхронные функции работали, нужно здесь поставить async и перед вложенными асинх.функциями await